target_chains/cosmwasm/contracts/pyth/src/wormhole.rs

```diff
@@ -1,3 +1,8 @@
+// These types are copied from the Wormhole contract. See the links with each type to see the original code
+// The reason to do so was dependency conflict. Wormhole contracts were using a very old version of a dependency
+// which is not compatible with the one used by osmosis-sdk. And since we weren't using anything else from
+// the Wormhole contract the types are moved here.
```

Comment:
this comment is more along the lines of what the other comments ^ should be like.